### ISSUE TITLE
Document display refresh rate

### ISSUE THREAD

**Issue by microbit-carlos** [lancaster-university/microbit-docs#58]
We had this question a few time already, it's very hard from the current docs to figure out how often the display is updated.

**Comment by microbit-sam** [comment]
The display is updated by the MicroBitDisplay `systemTick` function

I think that means that it updates every 6ms / 166.6 Hz, sound about right?

```
#define SYSTEM_TICK_PERIOD_MS                   6
```

**Comment by jamesadevine** [comment]
No. Each each row is strobed every 6 ms, which means that the display is strobed every 18 ms (55 Hz).

**Comment by microbit-sam** [comment]
Ah ok, thanks James!
